projects/v3/src/app/services/activity.service.ts

```diff
@@ -294,6 +294,8 @@ export class ActivityService {
     if (!getData) {
       return ;
     }
+
+    this.utils.setPageTitle(task.name);
```

Comment:
do you mean to set up the page title inside a `service` file?

Reply:
yes, if you check, this is the source of truth of every topic/assessment page (mobile and desktop) are using to access/switch task. You can trace back to see where it's being used in the component, this is done for DRY practice.